server/libs/userlibs.js

```diff
@@ -148,4 +150,100 @@ const logOut = function(request) {
     })
 }
 
-module.exports = {register, logIn, updateUser, logOut}
\ No newline at end of file
+const deleteUser = function(request) {
+    let id = request.params.uuid;
```

Comment:
deactive and delete will be two seperate functions and different routees witll be for them

Reply:
Done